server/user-update.js

```diff
@@ -63,6 +63,8 @@ module.exports = function handle({ message = {} }, { ship, hull }) {
     }
 
     if (errors && errors.length > 0) {
+      // TODO: this call can be easily too high volume:
+      // asUser.post(`/${ship.id}/notifications`, { status: "error", message: "Script error" });
```

Comment:
I'm strongly for the logs based solution here, we put some significant effort on having consistent logs done on single user level with all additional information to segment/aggregate. Want to list different outgoing.user.errors grouping them by %? With current information we have should be pretty easy:
```
1. outgoing.user.error - error: Wrong field mapping - 10% of users
2. outgoing.user.error - error: Email validation error - 1% of users
```

 Otherwise it looks like we would need to build on our own quite a infrastructure to handle big data here. I know that x-pack/ES based solution has it's hard points but at the end can we quickly build reliable notification ingestion?